tools/nxstyle.c

```diff
@@ -563,6 +563,7 @@ static const char *g_white_content_list[] =
   "unzGetCurrentFileInfo64",
   "unzGoToNextFile",
   "unzGoToFirstFile",
+  "XFree",
```

Comment:
I think this should move up to the section with the title `sim/posix/sim_x11framebuffer.c`. (This section right here is for `fs/zipfs/zip_vfs.c`.) Thanks.

Reply:
tools/nxstyle.c has been deleted, please ignore the false nxstyle alarm.